TYPO3.Neos/Resources/Private/TypoScript/Prototypes/Page.ts2

```diff
@@ -121,6 +121,17 @@ prototype(TYPO3.Neos:Page) < prototype(TYPO3.TypoScript:Http.Message) {
 		@if.onlyRenderWhenNotInLiveWorkspace = ${documentNode.context.inBackend}
 	}
 
+	neosBackendMetaData = TYPO3.TypoScript:Tag {
+		tagName = 'div'
+		@position = 'after neosBackendDocumentNodeData'
+		attributes = TYPO3.TypoScript:Attributes {
+			data-preview-uri = NodeUri {
+				node = ${q(documentNode).context({'workspaceName': documentNode.context.workspace.baseWorkspace.name}).get(0)}
+			}
+		}
+		@if.onlyRenderWhenNotInLiveWorkspace = ${documentNode.context.inBackend}
+	}
```

Comment:
makes sense, make it more robust.. did you try adding it to the existing DocumentMetadata?

Reply:
That was my first idea, but @grebaldi advised against it since DocumentMetadata is considered broken-ish. Maybe he can shed some light on this.